specs/spec_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'minitest'
+require 'minitest/spec'
+require 'minitest/autorun'
+require 'minitest/reporters'
+
+# require_relative '../far_mar'
```

Comment:
We should get this working; it would be better to have a single source of truth regarding which classes the module has. That way, when requirements change later, you're only changing them in one place instead of every spec file.